public/app/features/profile/api.ts

```diff
@@ -0,0 +1,57 @@
+import { getBackendSrv } from '@grafana/runtime';
+
+import { ChangePasswordFields, ProfileUpdateFields } from './types';
+import { Team, UserDTO, UserOrg, UserSession } from '../../types';
+
+async function changePassword(payload: ChangePasswordFields): Promise<void> {
+  try {
+    await getBackendSrv().put('/api/user/password', payload);
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+async function loadUser(): Promise<UserDTO> {
+  return await getBackendSrv().get('/api/user');
```

Comment:
```suggestion
  return getBackendSrv().get('/api/user');
```
Is there a reason we use `await` together with `return` in these functions? The only reason I see if we'd use them inside of `try/catch` clauses, but I even then we can just add `await` to the function itself.

Reply:
Hmm, I think I might have copied paste with little thought.

Reply:
Fixed!